docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -421,7 +421,7 @@ this person as a dependent.
     * If there is not a non-relative of the appropriate age available, assign to a non-relative who is older than 18 (select at random if multiple) 
     * If there are no non-relatives 18 or older, make "N/A"
 - Child is the reference person 
-    * If someone has a defined parent (or parent-in-law) relationship, assign them as guardian 
+    * If someone has a defined parent (or parent-in-law) relationship, assign them as guardian. If there are >2, assign 2 at random. 
```

Comment:
Throughout the rest of these rules, you describe choosing **one** guardian. Do you mean to choose 2 here? In that case, it becomes a little harder to interpret the section below "Once a guardian is assigned..."

Reply:
Fair! I changed to be just one assigned.